src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/BlobDispatcherTask.java

```diff
@@ -39,20 +37,10 @@ public BlobDispatcherTask(
     public void run() {
         logger.info("Started {} job", TASK_NAME);
 
-        List<String> containers = getAvailableContainers();
+        List<String> containers = serviceConfiguration.getEnabledSourceContainers();
```

Comment:
do we need to get the list every time?

Reply:
good point, I'll move it to constructor in next PR.